addons/website_profile/static/src/scss/website_profile.scss

```diff
@@ -249,3 +249,8 @@ $owprofile-color-bg: mix($body-bg, #efeff4);
 .o_wprofile_pointer {
     cursor: pointer;
 }
+
+// Own profile border
+.border_profile_focus {
+    border-left: 4px solid $secondary
+}
```

Comment:
- New class? Naming convention: o_appname_xxx. appname étant raccourci pour les website_ en w donc ici: o_wprofile_xxx
- Linter scss: missing ;
- For theme color variables, use the map as intended by BS4: `theme-color('secondary')`.

Reply:
What should the linter say (So I know if I correctly got one or not) ?

Reply:
> Linter scss: missing ;